persistence/src/jvmMain/kotlin/com/wire/kalium/persistence/kmmSettings/EncryptedSettingsHolder.kt

```diff
@@ -9,34 +8,33 @@ import java.io.FileWriter
 import java.nio.file.Paths
 import java.util.Properties
 
+private fun onModify(properties: Properties, file: File) {
+    properties.store(FileWriter(file), "Store values to properties file")
+}
+
+private fun createOrLoad(rootPath: String, file: File): Properties {
+    val properties = Properties()
+    File(rootPath).mkdirs()
+    if (!file.exists()) {
+        System.out.println(file.absolutePath)
+        file.createNewFile()
+    }
+    properties.load(FileInputStream(file))
```

Comment:
We have to close this `InputStream`, we can use the `.use` function to auto close it :D